code/style.css

```diff
@@ -1,58 +1,223 @@
+  font-size: 40px;
+  font-weight: bold;
+  text-align: center;
+  background: rgba(237, 228, 228, 0.5);
+  text-shadow: 1.5px 1px lightgray;
+}
+
+h2 {
+  align-self: center;
+  color: #fff;
+  font-style: italic;
+  text-align: center;
 }
 
 p {
-  margin: 1px;
+  margin: 0px;
   font-weight: 300;
   background-color: #fff;
 }
 
-.cards {
-  margin: 50px auto;
+/*** Styling of the header ***/
+header {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: center;
+}
+
+/*** Styling of the cards***/
+.cards { /* Outerbox containing desription and then a section with the decks. */
   width: 90%;
   height: 600px;
-  background: #ccc;
+  background: rgba(237, 228, 228, 0.3);
+}
+
+.cards-inner-box {
+  margin: 20px auto;
   display: flex;
   justify-content: space-around;
   align-items: center;
 }
 
-.killer-deck {
+.boldheading { /*Bold heading on the decks for 'Killer', 'Weapon', and 'Room'. */
+  font-weight: bold;
+}
+
+/* Killer-card */
+.killer-deck { 
   width: 250px;
-  height: 400px;
-  background: #000;
+  height: 440px;
+  background: #010f14;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
+}
+
+.killer-image {
+  margin: 15px 0px;
+  width: 150px;
+  height: auto;
+  border-radius: 16px;
 }
 
+/* Weapon-card */
 .weapon-deck {
   width: 250px;
-  height: 400px;
-  background: rgb(10, 53, 10);
+  height: 440px;
+  background: #78786c;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
+}
+
+.weapon-image {
+  width: 100px;
+  margin-top: 100px;
+  height: auto;
+  border-radius: 16px;
 }
 
+/* Room-card */
 .room-deck {
   width: 250px;
-  height: 400px;
-  background: midnightblue;
-
+  height: 440px;
+  background: #434254;
   text-align: center;
   padding-top: 30px;
   border-radius: 16px;
+  cursor: pointer;
 }
 
+.room-image {
+  width: 150px;
+  margin-top: 60px;
+  height: auto;
+  border-radius: 16px;
+}
+
+/* Styling of the loaders */
+.loader1, .loader2, .loader3 {
+  margin-left: 90px;
+  margin-top: 80px;
+  width: 60px;
+  height: 60px;
+  border: 10px solid #fff;
+  border-radius: 50%;
+  border-top: 10px solid #8c8e91;
+  -webkit-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
+  display: none;
+}
+
+@-webkit-keyframes spin { /* The animation of the loader */
+  0% { -webkit-transform: rotate(0deg); }
+  100% { -webkit-transform: rotate(360deg); }
+}
+
+@keyframes spin { /* The animation of the loader */
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+/** Styling of button **/
 button {
-  padding: 8px 16px;
-  background: crimson;
-  border: none;
+  margin: 20px;
+  padding: 8px 8px;
+  background: #fff; 
+  border: 1px solid #000;
+  box-shadow: 3px 3px 8px 3px grey;
   font-size: 20px;
   border-radius: 8px;
-  display: block;
+  display: flex;
   margin: 0 auto;
+  cursor: pointer;
+  font-family: 'Cormorant Garamond', serif;
 }
+
+button p {
```

Comment:
You can make a class and style it to make the code even clearer.

Reply:
Good suggestion